prusti-common/src/vir/mod.rs

```diff
@@ -4,21 +4,15 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-pub use self::ast::*;
-pub use self::cfg::*;
-pub use self::conversions::*;
 pub use self::to_viper::*;
 pub use self::program::*;
+pub use self::to_graphviz::*;
+pub use vir::legacy::*;
 
-mod ast;
-pub mod borrows;
-mod cfg;
-mod conversions;
 pub mod fixes;
 pub mod optimizations;
 mod to_viper;
-pub mod utils;
 mod program;
```

Comment:
Why program is not moved to `vir` crate?

Reply:
Mainly due to this: https://github.com/Maxwell-Yang-2001/prusti-dev/blob/polymorphic-vir-2/prusti-common/src/vir/optimizations/mod.rs#L49-L114, dependency on config as well as not as not possible to have "impl Program" when Program is not defined in current crate. Made a workaround in the new branch.